### PR TITLE
[WF] new job for daily builds and templates for additional tests

### PR DESCRIPTION
**Pull request by mbiarnes**
https://github.com/kiegroup/kie-jenkins-scripts/pull/1203
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1884

shortened job name

modified variable

parameter for Nexus URL

fixed execution of additional tests

removed obsolete baseBranch from additional tests

cron trigger new daily builds

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mareknovotny** [comment]
is this ready for approval? i think it looks good to me.
wdyt @ginxo?

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/JbpmContainerTestMatrix.groovy [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    static void addDeployConfiguration(MatrixJob matrixJob,
+                                       String kieVersion,
+                                       String jdkVersion,
+                                       String mvnTool,
+                                       String nexusUrl,
+                                       String settingsXml){
+
+        //Add configuration
+        matrixJob.with{
+            def jbpmContainerTest = '''#!/bin/bash -e
```
Comment (Ginxo) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/JbpmContainerTestMatrix.groovy:35]: this should be
```
            def jbpmContainerTest = '''#!/bin/bash -e
\n echo "KIE version $kieVersion"
\n echo "Nexus URL:  $nexusUrl"
\n wget -q $nexusUrl/org/jbpm/jbpm/$kieVersion/jbpm-$kieVersion-project-sources.tar.gz -O sources.tar.gz
\n tar xzf sources.tar.gz
\n rm sources.tar.gz
\n mv jbpm-$kieVersion/* .
\n rm -rf jbpm-$kieVersion'''
```
without `'''` content tabulation, or use
`\n rm -rf jbpm-$kieVersion'''.stripMargin()`
as it is done at line 46

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/JbpmTestCoverageMatrix.groovy [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    static void addDeployConfiguration(MatrixJob matrixJob,
+                                       String kieVersion,
+                                       String jdkVersion,
+                                       String mvnTool,
+                                       String nexusUrl,
+                                       String settingsXml){
+
+        //Add configuration
+        matrixJob.with{
+            def jbpmTestCoverage = '''#!/bin/bash -e
```
Comment (Ginxo) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/JbpmTestCoverageMatrix.groovy:35]: the same here. `.stripMargin()`

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/KieServerMatrix.groovy [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                                       String kieVersion,
+                                       String jdkVersion,
+                                       String mvnTool,
+                                       String downloadUrl,
+                                       String nexusUrl,
+                                       String settingsXml){
+
+        //Add configuration
+        matrixJob.with {
+            def kieServerTest = '''#!/bin/bash -e
```
Comment (Ginxo) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/KieServerMatrix.groovy:36]: the same here `.stripMargin()`

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/KieWbTestMatrix.groovy [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    static void addDeployConfiguration(MatrixJob matrixJob,
+                                       String kieVersion,
+                                       String jdkVersion,
+                                       String mvnTool,
+                                       String nexusUrl,
+                                       String settingsXml){
+
+        //Add configuration
+        matrixJob.with {
+            def kieWbTest = '''#!/bin/bash -e
```
Comment (Ginxo) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/KieWbTestMatrix.groovy:35]: the same here `.stripMargin()`

**Review by Ginxo — APPROVED** [review]
LGTM, just a minor thing

**Pull request merged**